app/generators/questionnaire_generator.rb

```diff
@@ -658,6 +662,36 @@ def generate_raw(question)
       question[:content].html_safe
     end
 
+    def generate_unsubscribe(question)
+      body = safe_join([
+                         generate_unsubscribe_content(question),
+                         generate_unsubscribe_action(question)
+                       ])
+      body = content_tag(:div, body, class: 'card light-grey-background-color')
+      body
+    end
+
+    def generate_unsubscribe_content(question)
+      body = []
+      body << content_tag(:span, question[:title].html_safe, class: 'card-title') if question[:title].present?
+      body << content_tag(:p, question[:content].html_safe) if question[:content].present?
+      body = safe_join(body)
+      body = content_tag(:div, body, class: 'card-content black-text')
+      body
+    end
+
+    def generate_unsubscribe_action(question)
+      response = Response.find_by_id(question[:response_id])
+      url_href = '#'
+      url_href = Rails.application.routes.url_helpers.questionnaire_path(uuid: response.uuid) if response
```

Comment:
Deze regel is trouwens wel een beetje jammer als we dit willen opsplitsen naar een service. Dan wil je dit eigenlijk ook injecten (met name als we dezelfde questionnaire service voor meerdere vsv instances gaan gebruiken). Maar dat is ook een probleem waar we dan naar kunnen gaan kijken.

Reply:
Ja hoe wou je dit dan doen? We zouden wel iets van een adapter-object ertussen kunnen bouwen, een soort van interface naar alle functies die we later eventueel willen vervangen. Zoals we dat destijds ook met de QubyService klasse hadden gedaan. Ik maak wel even een ticket aan. Maar dan merge ik dit eerst.